### PR TITLE
update links

### LINKED ISSUE #7: update links
http=>https

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/dwyl/learn-digitalocean/issues/7

### CODE REVIEW

**Comment by Arhell** [comment]
cc @nelsonic

File: .gitignore [hunk 1]
```diff
@@ -13,10 +13,10 @@ lib-cov
 # Coverage directory used by tools like istanbul
 coverage
 
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+# Grunt intermediate storage (httpы://gruntjs.com/creating-plugins#storing-task-files)
```
Comment (nelsonic) [inline .gitignore]:
```suggestion
# Grunt intermediate storage (https://gruntjs.com/creating-plugins#storing-task-files)
```
`ы` ?

File: .gitignore [hunk 2]
```diff
@@ -13,10 +13,10 @@ lib-cov
...
 .grunt
 
-# Compiled binary addons (http://nodejs.org/api/addons.html)
+# Compiled binary addons (httpы://nodejs.org/api/addons.html)
```
Comment (nelsonic) [inline .gitignore]:
```suggestion
# Compiled binary addons (https://nodejs.org/api/addons.html)
```

**Review by nelsonic — APPROVED** [review]
Thanks @Arhell 👍

**Comment by Arhell** [comment]
I'm sorry, I didn't change the keyboard layout and didn't notice the bug

**Pull request merged**